### DISCUSSION TITLE
How to insert sample data.

### DISCUSSION THREAD

**Discussion by kylehong1** [vendurehq/vendure#1914] [Q&A] [answered]
Hi I created new plugin and I want to add sample data on DB like populate.

How can I add the data?

**Comment by michaelbromley** [comment, marked as answer]
Hi,
Take a look at this guide: https://github.com/vendure-ecommerce/vendure/discussions/1912

That relates to adding Orders data, but the exact same principles apply to adding any kind of sample data.

**Reply by kylehong1** [reply]
Thanks, I successfully inserted my data.
